app/src/main/java/com/frinder/frinder/dataaccess/UserFirebaseDas.java

```diff
@@ -76,4 +88,60 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
             }
         });
     }
+
+    public interface UserDasInterface {
+        public void readUserComplete(User user);
+    }
+
+
+    @NonNull
+    private Map<String, Object> convertToFirebaseObject(User user) {
+        Map<String, Object> usr = new HashMap<>();
+        usr.put("id", user.getUid());
```

Comment:
nit: how about adding constant strings for these keys since we use this at multiple places.